.github/workflows/codeql.yml

```
@@ -0,0 +1,50 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: ["main"]
+  schedule:
+    - cron: '34 16 * * 3'
+
+jobs:
+  analyze:
+    name: Analyze (${{ matrix.language }})
+    runs-on: ubuntu-24.04
+    permissions:
+      # required for all workflows
+      security-events: write
```

that comment is now confusing actually, sorry for not spotting before. I guess it is meant to be meaningful in the context of the others you just removed, and someone would probably want to drop it / modify it when they adapt the template to their needs:

```suggestion
      security-events: write # to update code scanning alerts
```